subsys/input/input_lvgl_common.c

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2023 Fabian Blatz <fabianblatz@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <zephyr/device.h>
+#include <zephyr/input/input.h>
+#include <zephyr/kernel.h>
+#include <zephyr/drivers/display.h>
+
+#include "input_lvgl_common.h"
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(input_lvgl, CONFIG_INPUT_LOG_LEVEL);
```

Comment:
This may use a log level for the module rather than the input one, again, post code move comment.

Reply:
Changed to `LOG_MODULE_DECLARE(lvgl)`